samples/deploy/deployment/ingress-controller.yaml

```diff
@@ -41,7 +41,7 @@ spec:
       hostNetwork: true
       dnsPolicy: ClusterFirstWithHostNet
       containers:
-      - image: apache/apisix-ingress-controller:0.1.0
+      - image: apache/apisix-ingress-controller:1.0.0
```

Comment:
Please see the master branch， have already been 1.0.0 https://github.com/apache/apisix-ingress-controller/blob/master/samples/deploy/deployment/ingress-controller.yaml#L44